statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -57,165 +69,155 @@ def ma_rep(coefs, maxn=10):
 
     .. math:: y_t = \mu + \sum_{i=0}^\infty \Phi_i u_{t-i}
 
-    e.g. can recursively compute the \Phi_i matrices with \Phi_0 = I_k
-
-    Returns
-    -------
-    phis : ndarray (maxn + 1 x k x k)
+    The MA coefficients are :math:`\Phi_i` where :math:`\Phi_0 = I_k`.
     """
     p, k, k = coefs.shape
-    phis = np.zeros((maxn+1, k, k))
+    phis = np.zeros((maxn + 1, k, k))
     phis[0] = np.eye(k)
 
     # recursively compute Phi matrices
     for i in range(1, maxn + 1):
-        for j in range(1, i+1):
+        for j in range(1, i + 1):
             if j > p:
                 break
 
-            phis[i] += np.dot(phis[i-j], coefs[j-1])
+            phis[i] += np.dot(phis[i - j], coefs[j - 1])
 
     return phis
 
 
-def is_stable(coefs, verbose=False):
+@deprecate_kwarg('verbose', 'eigenvalues')
```

Comment:
Why this change?  `verbose` seems clearer to me.

Reply:
Oh I see; verbose and eigenvalues are doing different things.  In that case is deprecate_kwarg the right pattern?